package.json

```diff
@@ -49,6 +53,12 @@
       "import": "./prefixer/dist/goober-autoprefixer.mjs",
       "require": "./prefixer/dist/goober-autoprefixer.js",
       "umd": "./prefixer/dist/goober-autoprefixer.umd.js"
+    },
+    "./forwardProp": {
```

Comment:
How do you feel if we rename the `forwardProp` in `shouldForwardProp`? 🤔 it's longer, but might look more familiar for folks coming from other libraries.

Reply:
Yeah seems good to me!

Reply:
Do you want these reference to be changed everywhere (like in `forwardProp/package.json`) or just this "directory export"?

Reply:
How about everywhere for the sake of consistency? So the name should be camelCase (`shouldForwardProp`) and the folder should be dash-separated(`should-forward-prop`).

Reply:
Sounds good to me!